inskin-media/inskin-media-htb.js

```diff
@@ -0,0 +1,564 @@
+                dealId: bidDealId || null,
+                timeOfExpiry: expiry,
+                auxFn: __renderPixel,
+                auxArgs: [pixelUrl]
+            });
+
+            //? if (FEATURES.INTERNAL_RENDER) {
+            curReturnParcel.targeting.pubKitAdId = pubKitAdId;
+            //? }
+        }
+
+        if (hasBids) {
+            var win = window.top;
+            win.addEventListener('message', function (e) {
+                if (!e.data || e.data.from !== 'ism-bid') {
+                    return;
+                }
+
+                var decision = adResponse.decisions && adResponse.decisions[e.data.bidId];
+                if (!decision) {
+                    return;
+                }
+
+                var id = 'ism_tag_' + Math.floor(Math.random() * 10e16);
+                win[id] = {
+                    bidId: e.data.bidId,
+                    serverResponse: adResponse
+                };
+
+                var script = win.document.createElement('script');
+                script.src = 'https://cdn.inskinad.com/isfe/publishercode/' + bidsMap[e.data.bidId] + '/default.js?autoload&id=' + id;
+                win.document.getElementsByTagName('head')[0].appendChild(script);
+            });
+        }
+    }
+
+    /* =====================================
+     * Constructors
+     * ---------------------------------- */
+
+    (function __constructor() {
+        ComplianceService = SpaceCamp.services.ComplianceService;
+        RenderService = SpaceCamp.services.RenderService;
+
+        /* =============================================================================
+         * STEP 1  | Partner Configuration
+         * -----------------------------------------------------------------------------
+         *
+         * Please fill out the below partner profile according to the steps in the README doc.
+         */
+
+        /* ---------- Please fill out this partner profile according to your module ------------ */
+        __profile = {
+            partnerId: 'InskinMediaHtb',
+            namespace: 'InskinMediaHtb',
+            statsId: 'ISM',
+            version: '2.0.0',
+            targetingType: 'slot',
+            enabledAnalytics: {
+                requestTime: true
+            },
+            features: {
+                demandExpiry: {
+                    enabled: false,
+                    value: 0
+                },
+                rateLimiting: {
+                    enabled: false,
+                    value: 0
+                }
+            },
+
+            /* Targeting keys for demand, should follow format ix_{statsId}_id */
+            targetingKeys: {
+                id: 'ix_ism_id',
+                om: 'ix_ism_cpm',
+                pm: 'ix_ism_cpm',
+                pmid: 'ix_ism_dealid'
+            },
+
+            /* The bid price unit (in cents) the endpoint returns, please refer to the readme for details */
+            bidUnitInCents: 100,
```

Comment:
Your ad server is responding with 20. It looks like 20 cents instead of 20 dollars. Can you confirm the unit here? If it is 20 cents, update this line into `bidUnitInCents: 1,`. If not, leave this line as it is right now.

Reply:
Unit is dollars so no change necessary